spec/factories/event_schedule.rb

```diff
@@ -0,0 +1,22 @@
+# Read about factories at https://github.com/thoughtbot/factory_girl
+
+FactoryGirl.define do
+  factory :event_schedule do
+    event
+    after(:build) do |event_schedule|
```

Comment:
I think so. :pensive:
In some tests we only care about the room and in some other about the start_time, so we do not have to create it here. It has no sense creating a room without venue. And then if we are creating an event_schedule in the test is always because we are scheduling it, so it must belong to the selected_schedule, and if it doesn't exist it is created.